AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -61,6 +61,7 @@ report.az##.subs-in-social
 moemax.de##main > div[class] > div[data-testid="fullWidthComponent"] ~ div[class]:has(> div[class] > form[novalidate] button[data-purpose="newsletter.submit"])
 friendly2.me##.site-bottombar
 governing.com##.hbspt-form
+dtf.ru##.distribution
```

Comment:
1. Remove the rule
2. Add these rules because there will a leftover on mobile.
```adblock
dtf.ru#$#.distribution { display: none !important; }
!+ PLATFORM(android, ios)
dtf.ru#$#html { --layout-header-height-large: 60px !important; }
```